public/data/ChinmayMhatre.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Chinmay Mhatre",
+  "type": "personal",
+  "bio": "Developer | In a one-sided relationship with JavaScript",
+  "avatar": "https://github.com/ChinmayMhatre.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/ChinmayMhatre",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/chinmaysmhatre",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://linkedin.com/in/chinmaymhatre",
+      "icon": "linkedin"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "My blogs",
+      "icon": "book",
```

Comment:
There's a DEV.to icon. Would you like to add it? :)


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
Hey! thanks for letting me know. Will do that.